api/settings/read

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import sys
+import fileinput
+import subprocess
+import simplejson
+import os
+import re
+from datetime import datetime
+
+
+def get_config():
+    bash_command = "/sbin/e-smith/config getjson blacklist"
+    process = subprocess.Popen(bash_command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    return simplejson.loads(output)
+
+
+def get_confidence(category_id):
+    confidence_7_categories = ['firehol_level1']
+    confidence_6_categories = ['firehol_level2']
+
+    if category_id in confidence_7_categories:
+        return 7
+    elif category_id in confidence_6_categories:
+        return 6
+    else:
+        return 4
```

Comment:
Remove this function: do not make any exception for categories without confidence tag.